docs/_downloads/ee38530357642bb04ac0e6b7c2b97e61/plot_group_lasso.py

```diff
@@ -0,0 +1,219 @@
+# hand, thus we only show the left hemisphere. The activation is exactly in
+# the Primary somatosensory cortex. We highlight the borders of the post
+# central gyrus.
+
+
+t = 0.02
+t_idx = stcs[0].time_as_index(t)
+view = "lateral"
+
+for stc, subject in zip(stcs, subjects):
+    g_post_central = mne.read_labels_from_annot(subject, "aparc.a2009s",
+                                                subjects_dir=subjects_dir,
+                                                regexp="G_postcentral-lh")[0]
+    n_sources = [stc.vertices[0].size, stc.vertices[1].size]
+    m = abs(stc.data[n_sources[0], t_idx]).max()
+    surfer_kwargs = dict(
+        background="white", foreground="black",
+        clim=dict(kind='value', pos_lims=[0., 0.1 * m, m]),
+        hemi='lh', subjects_dir=subjects_dir,
+        initial_time=t * 1e3, time_unit='ms', size=(500, 500),
+        smoothing_steps=5, cortex=("gray", -1, 6, True))
+    brain = stc.plot(**surfer_kwargs, views=view)
+    brain.add_text(0.1, 0.9, subject + "_groupmne", "title")
+    brain.add_label(g_post_central, borders=True, color="green")
+
+#####################################
+# Group MNE leads to better accuracy
+# ----------------------------------
+# To evaluate the effect of the joint inverse solution, we compute the
+# individual solutions using `mne.inverse_sparse.mixed_norm` for each subject.
+# The group solutions are better located in S1.
+
+
+from mne.inverse_sparse import mixed_norm  # noqa: E402
+
+t = 0.02
+t_idx = stcs[0].time_as_index(t)
+view = "lateral"
+for subject, evoked, fwd, cov in zip(subjects, evokeds, fwds, noise_covs):
+    ev = evoked.copy()
+    ev.pick_types(meg="grad")
+    ev.crop(0.015, 0.025)
+    stc = mixed_norm(ev, fwd, cov, alpha=70., loose=0., depth=0.9)
+    stc.subject = subject
+    g_post_central = mne.read_labels_from_annot(subject, "aparc.a2009s",
+                                                subjects_dir=subjects_dir,
+                                                regexp="G_postcentral-lh")[0]
+    n_sources = [stc.vertices[0].size, stc.vertices[1].size]
+    m = abs(stc.data[:n_sources[0], t_idx]).max()
+    surfer_kwargs = dict(
+        background="white", foreground="black",
+        clim=dict(kind='value', pos_lims=[0., 0.1 * m, m]),
+        hemi='lh', subjects_dir=subjects_dir,
+        initial_time=t * 1e3, time_unit='ms', size=(500, 500),
+        smoothing_steps=5, cortex=("gray", -1, 6, True))
+    brain = stc.plot(**surfer_kwargs, views=view)
+    brain.add_text(0.1, 0.9, subject + "_mxne", "title")
+    brain.add_label(g_post_central, borders=True, color="green")
+
+
+###########################################
+# References
+# ----------
+# [1] Lim et al, Sparse EEG/MEG source estimation via a group lasso, PLOS ONE,
+# 2017
+#
+# [2] Jussi Nurminen, Hilla Paananen, & Jyrki Mäkelä. (2017). High frequency
+# somatosensory MEG: evoked responses, FreeSurfer reconstruction [Data set].
+# Zenodo. http://doi.org/10.5281/zenodo.889235
```

Comment:
all the files you generated in docs like the pngs should not be added to the repo. Just below...